platform/o.n.core/src/org/netbeans/core/NbURLDisplayer.java

```diff
@@ -105,6 +105,11 @@ private void warmBrowserUp( boolean externalBrowser ) {
         }
     }
 
+    @Override
+    public void clearCache() {
+        super.clearCache(); //To change body of generated methods, choose Tools | Templates.
+    }
+
```

Comment:
Why do you override a method that is doing nothing except call its super - which in turn does nothing?
Maybe setting htmlViewer to null should have been done here?

Reply:
You are right, my change was not uploaded, fixing it right now